buzzmobile/sense/inputer/inputer.lisp

```diff
@@ -0,0 +1,7 @@
+(in-package :buzzmobile)
+
+(defun inputer ()
+    (with-ros-node ("inputer")
```

Comment:
How does lisp know that ros is a thing it can access? Where is `with-ros-node` coming from/being imported?